### DISCUSSION TITLE
dde-dock: Dual Monitor

### DISCUSSION THREAD

**Discussion by Murilo2508** [linuxdeepin/developer-center#3047] [Features Request & Ideas | 特性请求 & 头脑风暴]
Is it possible to duplicate dock for dual monitor or multiple monitors?

**Comment by hugorn** [comment]
nice question!

**Comment by burkusmax** [comment]
That would be awesome!

**Comment by 0yanghongwei** [comment]
@Murilo2508 Hi. The dock can be moved in multi-screen mode. When the mouse stays at the bottom for a period of time, the dock will appear on the current screen.

**Comment by dreamtale90** [comment]
> @Murilo2508 Hi. The dock can be moved in multi-screen mode. When the mouse stays at the bottom for a period of time, the dock will appear on the current screen.

I always thought it was a bug for dock to jump back and forth between two monitors. Today I know it is a feature. How can this feature be turned off.

**Comment by meiyixiang** [comment]
There are no plans for that at the moment

**Comment by meiyixiang** [comment]
Hello, now in the latest version of the extended screen mode you can choose to follow the mouse position in the taskbar area of the screen to call up the taskbar display or on the main screen display so that the taskbar will only be displayed on the main screen, the two modes do not determine whether it meets your needs
